etsin_finder/qvain_light_file_service.py

```diff
@@ -0,0 +1,111 @@
+# This file is part of the Etsin service
+#
+# Copyright 2017-2018 Ministry of Education and Culture, Finland
+#
+# :author: CSC - IT Center for Science Ltd., Espoo Finland <servicedesk@csc.fi>
+# :license: MIT
+
+"""Used for performing operations related to Metax for Qvain Light"""
+
+import requests
+
+from etsin_finder.finder import app
+from etsin_finder.app_config import get_metax_api_config
+from etsin_finder.utils import json_or_empty, FlaskService
+
+log = app.logger
+
+class MetaxQvainLightAPIService(FlaskService):
```

Comment:
Qvain Light specific METAX API service class. Functionality was separated to its own file to make potential future separation of Qvain Light to its own app easier.